azure-toolkit-libs/azure-toolkit-auth-lib/pom.xml

```diff
@@ -63,6 +63,10 @@
             <groupId>com.microsoft.azure</groupId>
             <artifactId>azure-client-runtime</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.azure.resourcemanager</groupId>
```

Comment:
why add resource manager dependency here?

Reply:
auth-lib depends on this, but not declared.